designcompose/src/main/java/com/android/designcompose/DocServer.kt

```diff
@@ -66,7 +67,32 @@ internal class LiveDocSubscriptions(
     val subscribers: ArrayList<LiveDocSubscription> = ArrayList(),
 )
 
+/**
+ * Design doc status
+ *
+ * Publicly accessible status for a DesignDoc. Accessible via DesignSettings.designDocStatuses
+ */
+class DesignDocStatus() {
+    // If set, the time when the doc was last loaded from storage. Otherwise, this has not happened
+    var lastLoadFromDisk: Instant? = null
```

Comment:
Do we need times for these? Would a count work OK?

Reply:
ok, that makes sense (but can be an internal API in that case)

Reply:
It could be. I've made the setters internal so it's essentially read only. Do you think it's worth hiding that information?